cross-make-mingw64.sh

```diff
@@ -1,24 +1,24 @@
 #!/bin/sh
 
+# 32 and 64 bits builds of mingw for building 64 bits windows binaries
 CMD_PREFIX="amd64-mingw32msvc x86_64-w64-mingw32";
 
-if [ "X$CC" = "X" ]; then
-    for check in $CMD_PREFIX; do
-        full_check="${check}-gcc"
+# Define CC to the correct (32 or 64 bits) version of mingw
+for check in $CMD_PREFIX; do
+	full_check="${check}-gcc"
 	if [ ! $(which "$full_check") = "" ]; then
-	    export CC="$full_check"
+		export CC="$full_check"
+		MINGW_VERSION=${check};
 	fi
-    done
-fi
+done
 
-if [ "X$WINDRES" = "X" ]; then
-    for check in $CMD_PREFIX; do
-        full_check="${check}-windres"
+# Define WINDRES to the correct (32 or 64 bits) version of mingw
+for check in $CMD_PREFIX; do
```

Comment:
Can this be combined into the first loop?

Reply:
Fixed